velox/exec/fuzzer/JoinFuzzer.cpp

```diff
@@ -1153,7 +1206,7 @@ void JoinFuzzer::verify(core::JoinType joinType) {
     }
 
     if (FLAGS_enable_spill) {
-      // Spilling for right semi project doesn't work yet.
+      // Spilling for right semi project doesn't work yet.ƒverify()
```

Comment:
nit: What's this "ƒverify()" for?

Reply:
I don't know how that got in there. Removed.